docs/technical_reference/flowsheets/BSM2.rst

```diff
@@ -37,6 +37,15 @@ The flowsheet relies on the following key assumptions:
 
     Figure 1. BSM2 flowsheet
 
+The objective function is to minimize the levelized cost of water, which can be represented by the following equation
+where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
+:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
+:math:`f_{util}` represents the utilization factor:
+
+    .. math::
+
+        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
+
```

Comment:
Add a documentation for unit costing section

Reply:
See lines 54-64 in the BSM2 flowsheet documentation. I've moved the WaterTAP costing package link to this sectoin as well as the objective function.